sentry/src/main/java/io/sentry/SentryOptions.java

```
@@ -537,13 +540,27 @@ public void addIntegration(@NotNull Integration integration) {
     return dsn;
   }
 
+  /**
+   * Returns the DSN
+   *
+   * @return the DSN or null if not set
```

Should we document here that it might throw if the DSN is invalid?